modules/odc-creating-apiserversource.adoc

```diff
@@ -31,6 +27,11 @@ image::apiserver-odc-config-1.png[Select the Service Account Name and Sink]
 +
 image::toplogy-odc-apiserver.png[ApiServerSource Topology view]
 
+[NOTE]
+====
+If an **URI sink** is used, the URI can be modified by right clicking on the **URI sink** -> `Edit URI`.
```

Comment:
Not sure if "an" or "a" URI sink. My vote would be "a" but I know others for OCP docs may have a different opinion. Just a suggestion, feel free to ignore.